base/logger_file.go

```diff
@@ -116,7 +116,11 @@ func NewFileLogger(config *FileLoggerConfig, level LogLevel, name string, logFil
 func (l *FileLogger) FlushBufferToLog() {
```

Comment:
Super-nit but slightly earlier return opportunity

```go
func (l *FileLogger) FlushBufferToLog() {
	if !l.Enabled.IsTrue() {
		return
	}

	// Need to clear hanging new line to avoid empty line
	logString := strings.TrimSuffix(l.buffer.String(), "\n")
	l.logf(logString)
}
```

Reply:
Don't think this is worth based on discussed changes to add reset.